docs/source/science/GEDI/GEDI_L4A.ipynb

```diff
@@ -2,11 +2,57 @@
  "cells": [
```

Comment:
<p>this seems to have rendered differently, might be an extra space before the first bullet?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/270/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>